addons/mail/static/src/components/follower/follower.xml

```diff
@@ -4,7 +4,7 @@
     <t t-name="mail.Follower" owl="1">
         <div class="o_Follower">
             <t t-if="follower">
-                <a class="o_Follower_details" t-att-class="{ 'o-inactive': !follower.isActive }" href="#" role="menuitem" t-on-click="_onClickDetails">
+                <a class="o_Follower_details" t-att-class="{ 'o-inactive': !follower.isActive, 'py-1': !follower.isEditable }" href="#" role="menuitem" t-on-click="_onClickDetails">
```

Comment:
![Screenshot from 2023-05-10 14-35-19](https://github.com/odoo/odoo/assets/99004966/b6d01e85-9825-4e16-8bc3-52a2bb67cb5b)

As you can see the `py-1` is for the padding issue between two followers.

Reply:
And why limit to non-editable follower?
I suppose it's not a problem with editing because icons do some padding.
If it's good, I would always put `py-1`. The conditional on `!isEditable` looks off

Reply:
Indeed the Icons adds some padding between followers. That's why the padding is for only non-editable followers. And adding unconditional `py-1` would add extra unnecessary padding around editable follower. 

But i would try to figure out something else and if the change won't be too much I would change it.

Reply:
Looks like you can add it even when editable, so can just put `py-1` all the time